Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Basic docker image for RocketMap
+# Usage:
+#   docker build -t rocketmap .
+#   docker run -d -P rocketmap -a ptc -u YOURUSERNAME -p YOURPASSWORD -l "Seattle, WA" -st 10 --gmaps-key CHECKTHEWIKI
+
+FROM python:3.6
```

Comment:
MIght aswell run on 3.7 as in the README it says to use it (although other 3.x are fine :)